.github/workflows/ghcr.yml

```
@@ -26,16 +26,79 @@ permissions:
+
 jobs:
-  publish:
-    name: Build and push to GHCR
-    runs-on: ubuntu-latest
+  build:
+    name: Build ${{ matrix.platform }}
+    runs-on: ${{ matrix.platform == 'linux/amd64' && 'ubuntu-latest' || 'ubuntu-24.04-arm' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ linux/amd64, linux/arm64 ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)